types/node/globals.d.ts

```diff
@@ -234,6 +234,20 @@ declare global {
              * Is this a constructor call?
              */
             isConstructor(): boolean;
+
+            getScriptHash(): string;
+            getScriptNameOrSourceURL(): string;
+
+            getScriptNameOrSourceURL(): string;
+            getEnclosingColumnNumber(): number;
+            getEnclosingLineNumber(): number;
+            getPosition(): number;
+
+            getPromiseIndex(): number;
```

Comment:
Looks like it can return `null`
https://github.com/nodejs/node/blob/ac9e594e3224363691e1d5aecd291af3137d832a/deps/v8/src/builtins/builtins-callsite.cc#L126-L129

Reply:
Thanks. Return type changed to `number | null`